store/src/java/com/zimbra/cs/service/FileUploadServlet.java

```diff
@@ -881,10 +881,52 @@ private static long getFileUploadMaxSize(boolean limitByFileUploadMaxSize) {
         return maxSize;
     }
 
+    private static long getFileUploadMaxSize(boolean limitByFileUploadMaxSize, Account acct) {
+        long maxSize = DEFAULT_MAX_SIZE;
+        String attributeUsed = null;
+        try {
+            if (limitByFileUploadMaxSize && acct.getFileUploadMaxSizePerFile() > 0) {
+                long fileUploadSizePerFile = acct.getFileUploadMaxSizePerFile();
```

Comment:
Move the variable creation inside respective if and else condition, so that at least one variable creation step can be saved in each execution.